RMS/BufferedCapture.py

```diff
@@ -1081,6 +1187,47 @@ def run(self):
                     first_frame_timestamp = frame_timestamp
 
 
+                # If save_frames is set and a video device is used, save a frame every nth frames
+                if (self.config.save_frames
+                    and self.video_file is None
+                    and total_frames % self.config.frame_save_interval_count == 0):
+
+                    # reset start time values everytime the buffers are switched
+                    if self.raw_frame_count == 0:
+
+                        if raw_buffer_one:
+                            self.startRawTime1.value = 0
+                        else:
+                            self.startRawTime2.value = 0
+
+                        # Always set first raw frame timestamp in the beginning of the block
+                        first_raw_frame_timestamp = frame_timestamp 
+
+
+                    # Write raw frame and timestamp to one of the two corresponding buffers
+                    if raw_buffer_one:
+                        self.sharedRawArray[self.raw_frame_count, :, :] = frame
+                        self.sharedTimestamps[self.raw_frame_count] = frame_timestamp
+
+                    else:
+                        self.sharedRawArray2[self.raw_frame_count, :, :] = frame
+                        self.sharedTimestamps2[self.raw_frame_count] = frame_timestamp
+                    

```

Comment:
I'm getting this error when attempting to capture during the day:
"ValueError: could not broadcast input array from shape (1080,1920,3) into shape (1080,1920)"
This error occurs because we're trying to write a 3-channel color frame (shape 1080x1920x3) into a 2D array buffer (shape 1080x1920).